var/spack/repos/builtin/packages/kaldi/0001-Fix-CUDA_INCLUDE_DIRS.patch

```diff
@@ -0,0 +1,27 @@
+From 5a9f3c8fd7c132b1591bb34b85bebc59d4dfcb24 Mon Sep 17 00:00:00 2001
+From: Roland Fehrenbacher <rf@q-leap.de>
+Date: Thu, 25 Jan 2024 20:04:43 +0100
+Subject: [PATCH 1/2] Fix CUDA_INCLUDE_DIRS
+
+---
+ CMakeLists.txt | 4 +---
+ 1 file changed, 1 insertion(+), 3 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 886af19c6..89a178b9b 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -198,9 +198,7 @@ if(CUDA_FOUND)
+     include_directories(${NvToolExt_INCLUDE_DIR})
+     link_libraries(${NvToolExt_LIBRARIES})
+ 
+-
```

Comment:
Are `cub` and `cuda` mutually exclusive?  

Is `cub` supported anymore? If not, then the `cub` dependency should probably be tied to older versions where it is supported (see https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-dependencies).

Reply:
I guess cub is still needed for cuda <= 10 as the dep line states, but I can't tell for sure. I left it in not to break builds with older CUDA.